entity-management-definitions/src/main/java/eu/europeana/entitymanagement/utils/UriValidator.java

```diff
@@ -0,0 +1,50 @@
+package eu.europeana.entitymanagement.utils;
+
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+
+public class UriValidator {
+
+  private static final Predicate<String> URN_MATCH_PREDICATE =
+      Pattern.compile(
+              "^urn:[a-z0-9][a-z0-9-]{0,31}:([a-z0-9()+,\\-.:=@;$_!*']|%[0-9a-f]{2})+$",
+              Pattern.CASE_INSENSITIVE)
+          .asMatchPredicate();
+
+  // only http / https schemes supported, non-latin characters should be percent-encoded
+  private static final Predicate<String> URL_MATCH_PREDICATE =
+      Pattern.compile("^(https?)://[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]")
```

Comment:
To support additional schemes (eg ftp / ftps), `(https?)` can be changed to `(https?|ftps?)`.